test/mocha.opts

```diff
@@ -0,0 +1 @@
+-r jsdom-global/register
```

Comment:
I don't think you need this if you are setting things up in the `_setup.js` file.